patches/chrome-browser-ui-views-tabs-tab.cc.patch

```diff
@@ -10,6 +10,17 @@ index 14704ed4ed53e034f4d43bd30330c486fc64bc67..7d81149a1f20efcc6c5a348fdc714d87
      } else if (showing_close_button_) {
        // Allow the title to overlay the close button's empty border padding.
        title_right = close_x - after_title_padding;
+@@ -995,8 +996,8 @@ void Tab::UpdateIconVisibility() {
+   showing_icon_ = showing_alert_indicator_ = false;
+   extra_alert_indicator_padding_ = false;
+ 
+-  if (height() < GetLayoutConstant(TAB_HEIGHT))
+-    return;
++  // if (height() < GetLayoutConstant(TAB_HEIGHT))
++  //  return;
```

Comment:
This early return causes icons to be hidden if tab dimensions differ in different modes (e.g. horizontal/vertical). It's not clear what the purpose of this early return might be.

Reply:
Thanks! this looks good to me as is but wonder what you think about this

```c++
chromium_src/ ... / tab.cc

#include "brave/browser/ui/views/tabs/vertical_tab_utils.h"
#include "chrome/browser/ui/layout_constants.h"

...

#define GetLayoutConstant(COMPONENT)                                 \
  ((COMPONENT == TAB_HEIGHT &&                                       \
    tabs::utils::ShouldShowVerticalTabs(controller()->GetBrowser())) \
       ? tabs::kVerticalTabHeight                                    \
       : GetLayoutConstant(COMPONENT))

#include "src/chrome/browser/ui/views/tabs/tab.cc"

#undef GetLayoutConstant


```

Could this be better? Totally optional opinion, so it's up to you.

Reply:
This works great - thanks!